ci/jjb/jobs/pulp-3-installer.yaml

```diff
@@ -0,0 +1,55 @@
+# Install Pulp 3 on a host.
+- job:
+    name: pulp-3-installer
+    concurrent: true
+    node: 'fedora-np'
+    description: |
+      <p>
+        Install Pulp on the host identified by the job parameter
+        <code>HOSTNAME</code>.
+      </p>
+      <p>
+        Ensure the target host has the following line in
+        <code>~/.ssh/authorized_keys</code>:
+      </p>
+      <pre>
+        ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC6DJ8fmd61DWPCMiOEuy96ajI7rL3rWu7C9NQhE9a4SfyaiBcghREHJNCz9LGJ57jtOmNV0+UEDhyvTckZI2YQeDqGCP/xO9B+5gQNlyGZ9gSmFz+68NhYQ0vRekikpb9jNdy6ZZbfZDLp1w7dxqDIKfoyu7QO3Qr3E/9CpiucQif2p+oQOVOCdKEjvGYNkYQks0jVTYNRscgmcezpfLKhqWzAre5+JaMB0kRD5Nqadm2uXKZ4cNYStrpZ4xUrnMvAqjormxW2VJNx+0716Wc2Byhg8Nva+bsOkxp/GewBWHfNPtzQGMsL7oYZPtOd/LrmyYeu/M5Uz7/6QCv4N90P pulp
+      </pre>
+      <p>
+        In addition, ensure the target host satisfies the <a
+        href="http://docs.ansible.com/ansible/latest/intro_installation.html#managed-node-requirements">managed
+        node requirements</a>.
+      </p>
+    parameters:
+      - string:
+          name: HOSTNAME
+      - string:
+          name: USER
+          default: root
+    properties:
+      - qe-ownership
+    scm:
+      - git:
+          url: https://github.com/pulp/devel.git
+          branches:
+            - '3.0-dev'
+          skip-tag: true
+    wrappers:
+      - jenkins-ssh-credentials
+    builders:
+      - shell: |
+          sudo yum install -y ansible python3
+          cd ansible
+          ansible-galaxy install -r requirements.yml -p roles
+          export ANSIBLE_HOST_KEY_CHECKING=False
+          ansible all \
+            --inventory "$HOSTNAME", \
+            --private-key ../pulp_server_key \
```

Comment:
I have no idea where this file comes from. It's magical. Perhaps the [jenkins-ssh-credentials](https://github.com/pulp/pulp_packaging/blob/master/ci/jjb/jobs/macros.yaml#L213-L226) macro creates it? The existing [pulp-installer](https://github.com/pulp/pulp_packaging/blob/master/ci/jjb/jobs/pulp-installer.yaml#L60) job also references this file, which is why I'm referencing it here.